system/jlib/jqueue.tpp

```diff
@@ -557,7 +557,6 @@ public:
 
     ~ReallySimpleInterThreadQueueOf<BASE, ALLOWNULLS>()
     {
-        stop();
```

Comment:
This stop is unnecessary, and would never be thread safe.  If something was waiting it could access memory after it was freed.